ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activityDateAnswer.component.ts

```diff
@@ -33,14 +33,16 @@ export class ActivityDateAnswer {
     constructor(private logger: LoggingService) { }
 
     public handleChange(value: DatePickerValue | null): void {
-        if (value == null) {
-            return;
+        let dateValue: DatePickerValue | null = value;
```

Comment:
This assignment is a bit odd. Intent clearer perhaps if just:
```suggestion
        let dateValue: DatePickerValue | null = null;
```